src/common/email.ts

```diff
@@ -0,0 +1,31 @@
+export async function sendEmail(
+    to: string[],
+    subject: string,
+    template_id: string,
+    template_data?: object,
+    bcc?: string[]
+) {
+    console.log(`Sending email to ${to} with ${bcc} with template ${template_id}`);
+
+    if (process.env.NODE_ENV === "development" && !process.env.SMTP2GO_KEY) {
```

Comment:
making @wingio's comment a suggestion
```suggestion
import { prod } from "./constants";

export async function sendEmail(
    to: string[],
    subject: string,
    template_id: string,
    template_data?: object,
    bcc?: string[]
) {
    console.log(`Sending email to ${to} with ${bcc} with template ${template_id}`);

    if (!prod && !process.env.SMTP2GO_KEY) {
```